docs/sql/queries.txt

```diff
@@ -48,9 +48,10 @@ Aliases can be used to change the output name of the columns::
 FROM Clause
 -----------
 
-The ``FROM`` clause is used to reference the table to query. A table is referenced
-by schema and table name and can optionally be aliased. If the table ``t`` is
-only referenced by table name, crate assumes the table ``doc.t`` was meant.
+The ``FROM`` clause is used to reference one or two tables.
```

Comment:
... to reference the relation this select query is based upon. Can be a single table, many tables or a table :ref:`JOIN <sql_joins>`.